action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'NPM Version Incrementer'
+description: 'Increments the version of an NPM package based on the branch.'
+outputs:
+  branch-name:
+    description: 'The current branch name.'
+    value: ${{ steps.get-branch-name.outputs.branch-name }}
+  current-version:
+    description: 'The current version of the package.'
+    value: ${{ steps.get-current-version.outputs.current-version }}
+  new-version:
+    description: 'The new version of the package.'
+    value: ${{ steps.get-new-version.outputs.new-version }}
+runs:
+  using: "composite"
+  steps:
+    - id: get-branch-name
+      shell: bash
+      run: |
+          # Get the current branch name
+          BRANCH=$(git branch --show-current)
+          echo "Current branch is - $BRANCH"
+          echo "branch-name=$BRANCH" >> $GITHUB_OUTPUT
```

Comment:
I don't think we need branch-name output.

Reply:
I included this as a way of confirming that the correct branch is being built, and so that you can confirm that it's building major/minor/patch as expected. I can remove it if you don't think it will be useful.

Reply:
I don't see anywhere it's used. And [GITHUB_REF](https://docs.github.com/en/actions/learn-github-actions/variables) is for that.